### ISSUE TITLE
Conflict with `flake8-bandit` plugin

### ISSUE THREAD

**Issue by ndtho8205** [snok/flake8-type-checking#74]
**Describe the bug**

Conflict with the `flake8-bandit` plugin. To the best of my understanding, I think the bug can be fixed in `flake8-type-checking`, so I create a new issue here instead.

**To Reproduce**

test.py

```python
a.a = 20
```

pyproject.toml:

```toml
[tool.poetry]
name = "test"
version = "0.1.0"
description = ""
authors = ["Noname <noname@email.com>"]

[tool.poetry.dependencies]
python = "^3.8"
flake8 = "^4.0.1"
flake8-type-checking = "^1.3.2"
flake8-bandit = "^3.0.0"

[tool.poetry.dev-dependencies]

[build-system]
requires = ["poetry-core>=1.0.0"]
build-backend = "poetry.core.masonry.api"
```

Run flake8 with these arguments:

```sh
flake8 test.py
```

The resulting error is:

```
Traceback (most recent call last):
  File "/home/noname/.cache/pypoetry/virtualenvs/type-cheking-bug-ZJCJNDYT-py3.8/bin/flake8", line 8, in <module>
    sys.exit(main())
  File "/home/noname/.cache/pypoetry/virtualenvs/type-cheking-bug-ZJCJNDYT-py3.8/lib/python3.8/site-packages/flake8/main/cli.py", line 22, in main
    app.run(argv)
  File "/home/noname/.cache/pypoetry/virtualenvs/type-cheking-bug-ZJCJNDYT-py3.8/lib/python3.8/site-packages/flake8/main/application.py", line 375, in run
    self._run(argv)
  File "/home/noname/.cache/pypoetry/virtualenvs/type-cheking-bug-ZJCJNDYT-py3.8/lib/python3.8/site-packages/flake8/main/application.py", line 364, in _run
    self.run_checks()
  File "/home/noname/.cache/pypoetry/virtualenvs/type-cheking-bug-ZJCJNDYT-py3.8/lib/python3.8/site-packages/flake8/main/application.py", line 271, in run_checks
    self.file_checker_manager.run()
  File "/home/noname/.cache/pypoetry/virtualenvs/type-cheking-bug-ZJCJNDYT-py3.8/lib/python3.8/site-packages/flake8/checker.py", line 311, in run
    self.run_serial()
  File "/home/noname/.cache/pypoetry/virtualenvs/type-cheking-bug-ZJCJNDYT-py3.8/lib/python3.8/site-packages/flake8/checker.py", line 295, in run_serial
    checker.run_checks()
  File "/home/noname/.cache/pypoetry/virtualenvs/type-cheking-bug-ZJCJNDYT-py3.8/lib/python3.8/site-packages/flake8/checker.py", line 597, in run_checks
    self.run_ast_checks()
  File "/home/noname/.cache/pypoetry/virtualenvs/type-cheking-bug-ZJCJNDYT-py3.8/lib/python3.8/site-packages/flake8/checker.py", line 500, in run_ast_checks
    for (line_number, offset, text, _) in runner:
  File "/home/noname/.cache/pypoetry/virtualenvs/type-cheking-bug-ZJCJNDYT-py3.8/lib/python3.8/site-packages/flake8_type_checking/plugin.py", line 81, in run
    visitor = TypingOnlyImportsChecker(self._tree, self.options)
  File "/home/noname/.cache/pypoetry/virtualenvs/type-cheking-bug-ZJCJNDYT-py3.8/lib/python3.8/site-packages/flake8_type_checking/checker.py", line 568, in __init__
    self.visitor.visit(node)
  File "/home/noname/.pyenv/versions/3.8.12/lib/python3.8/ast.py", line 371, in visit
    return visitor(node)
  File "/home/noname/.pyenv/versions/3.8.12/lib/python3.8/ast.py", line 447, in generic_visit
    value = self.visit(value)
  File "/home/noname/.pyenv/versions/3.8.12/lib/python3.8/ast.py", line 371, in visit
    return visitor(node)
  File "/home/noname/.cache/pypoetry/virtualenvs/type-cheking-bug-ZJCJNDYT-py3.8/lib/python3.8/site-packages/flake8_type_checking/checker.py", line 380, in visit_Assign
    self.generic_visit(node)
  File "/home/noname/.pyenv/versions/3.8.12/lib/python3.8/ast.py", line 447, in generic_visit
    value = self.visit(value)
  File "/home/noname/.pyenv/versions/3.8.12/lib/python3.8/ast.py", line 371, in visit
    return visitor(node)
  File "/home/noname/.cache/pypoetry/virtualenvs/type-cheking-bug-ZJCJNDYT-py3.8/lib/python3.8/site-packages/flake8_type_checking/checker.py", line 442, in visit_Attribute
    node = self._set_child_node_attribute(node, ATTRIBUTE_PROPERTY, node.attr)
  File "/home/noname/.cache/pypoetry/virtualenvs/type-cheking-bug-ZJCJNDYT-py3.8/lib/python3.8/site-packages/flake8_type_checking/checker.py", line 426, in _set_child_node_attribute
    setattr(node.__dict__[key], attr, val)
AttributeError: 'tuple' object has no attribute 'flake8-type-checking_parent'
```

**Expected behavior**

The above command should return no error. When I removed the `flake8-bandit` plugin, it worked as expected.

**Environment**

- OS and Python version: Linux/Python 3.8.12
- `flake --bug-report` output:
  ```json
  {
    "dependencies": [],
    "platform": {
      "python_implementation": "CPython",
      "python_version": "3.8.12",
      "system": "Linux"
    },
    "plugins": [
      {
        "is_local": false,
        "plugin": "flake8-bandit",
        "version": "3.0.0"
      },
      {
        "is_local": false,
        "plugin": "flake8-type-checking",
        "version": "1.3.2"
      },
      {
        "is_local": false,
        "plugin": "mccabe",
        "version": "0.6.1"
      },
      {
        "is_local": false,
        "plugin": "pycodestyle",
        "version": "2.8.0"
      },
      {
        "is_local": false,
        "plugin": "pyflakes",
        "version": "2.4.0"
      }
    ],
    "version": "4.0.1"
  }
  ```

**Additional context**

I created a breakpoint before line 457 in the `checker.py` file.

```python
    @staticmethod
    def _set_child_node_attribute(node: Any, attr: str, val: Any) -> Any:
        # Set the parent attribute on the current node children
        for key, value in node.__dict__.items():
            if type(value) not in [int, str, list, bool] and value is not None:
                __import__("pdb").set_trace()
                setattr(node.__dict__[key], attr, val)
        return node
```

The following is the information that I can gather. Hope it helps you.

```
> /home/noname/.cache/pypoetry/virtualenvs/type-cheking-bug-ZJCJNDYT-py3.8/lib/python3.8/site-packages/flake8_type_checking/checker.py(457)_set_child_node_attribute()
-> setattr(node.__dict__[key], attr, val)
(Pdb) cont

> /home/noname/.cache/pypoetry/virtualenvs/type-cheking-bug-ZJCJNDYT-py3.8/lib/python3.8/site-packages/flake8_type_checking/checker.py(456)_set_child_node_attribute()
-> __import__("pdb").set_trace()
(Pdb) cont

> /home/noname/.cache/pypoetry/virtualenvs/type-cheking-bug-ZJCJNDYT-py3.8/lib/python3.8/site-packages/flake8_type_checking/checker.py(457)_set_child_node_attribute()
-> setattr(node.__dict__[key], attr, val)
(Pdb) cont

> /home/noname/.cache/pypoetry/virtualenvs/type-cheking-bug-ZJCJNDYT-py3.8/lib/python3.8/site-packages/flake8_type_checking/checker.py(456)_set_child_node_attribute()
-> __import__("pdb").set_trace()
(Pdb) key
'_bandit_linerange'

(Pdb) node.__dict__[key]
(1, 1)

(Pdb) dir(node.__dict__[key])
['__add__', '__class__', '__contains__', '__delattr__', '__dir__', '__doc__', '__eq__', '__format__', '__ge__', '__getattribute__', '__getitem__', '__getnewargs__', '__gt__', '__hash__', '__init__', '__init_subclass__', '__iter__', '__le__', '__len__', '__lt__', '__mul__', '__ne__', '__new__', '__reduce__', '__reduce_ex__', '__repr__', '__rmul__', '__setattr__', '__sizeof__', '__str__', '__subclasshook__', 'count', 'index']

(Pdb) getattr(node.__dict__[key], ATTRIBUTE_PROPERTY)
*** AttributeError: 'tuple' object has no attribute 'flake8-type-checking_parent'

(Pdb)
```

**Comment by sondrelg** [comment]
Huh, is flake8-bandit pruning ast node attributes in some way?

I'm away for a few days; would you be interested in trying to create a fix?

**Comment by ndtho8205** [comment]
I'm sorry but I can't help since I don't have any idea about how your plugin and flake8-bandit work.

**Comment by sondrelg** [comment]
Think the problem should be fixed in v1.3.3. Would you mind testing whether upgrading fixes the problem for you @ndtho8205?

**Comment by ndtho8205** [comment]
The bug was fixed. Thank you for your great effort and your great plugin too.
